### ISSUE TITLE
App Slowness reported Since upgrade of 4.6

### ISSUE THREAD

**Issue by rezzap** [woocommerce/woocommerce-android#2869]
**Describe the bug**

We've been seeing an increase in reports about app slowness since version 4.6. This is especially significant when trying to load orders. 

A user has reported this and shown how significantly different this is between version 4.6 and 4.9 in a video reported in 3319401-zen. Since we've been seeing an increase of these reports in app reviews and tickets, I'm opening this issue to have a better place to keep track of them. 

**To Reproduce**
1. Go to My Store > Orders in version 4.6 of the app.
2. Click on a new order
3. Wait for the order to load and notice the speed. 
4. Go back to the Play Store and Update the app
5. Come back to the app and go back to My Store > Orders in the latest version of the app
4. Notice how much slower the app loads the content of the order and it's images. 

**Screenshots**
Video available in the ticket. 

**Expected behavior**
I wouldn't expect to see such a significant difference between app versions. 

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**Mobile Environment**

I'm asking the user to provide more information and will add it to the report once received. 
Please include:
- Device: 
- Android version: 
- WooCommerce Android version: 4.7 and up

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by erricgunawan** [comment]
User on 3319401-zen replied with this Device info:

> Device Mi 9T Pro, miui 12 based on android 10.

They also sent their System Status Report.

**Comment by nbradbury** [comment]
I've been looking into this, and I believe I've found the problem. I'll submit a PR soon and will respond here when it's merged.

**Comment by nbradbury** [comment]
@erricgunawan This has been fixed and the fix will be in v5.3 of the app.
